### ISSUE TITLE
I want to add svm in last layer of my model

### ISSUE THREAD

**Issue by feridesecil13** [keras-team/keras#19078]
Hello
I am extracting buildings from 512x512 images using the Massachusetts building dataset. In order to get more accurate results, I want to add a Support Vector Machine to the last layer of the architecture I created, that is, the classifier layer. I don't want to apply Flatten because I aim to get 512x512 output dimensions. Can you help me?

The last part of my architecture is like this;

    c7=(BatchNormalization())(c7)
    u7=(Conv2DTranspose(32,(3,3), strides=(2, 2), activation='relu', kernel_initializer='he_normal', padding='same'))(c7)
    u7=(Dropout(0.3))(u7)
    c8=(Conv2D(32, (3,3),activation='relu', kernel_initializer='he_uniform', padding='same'))(u7)
    c8=(Dropout(0.2))(c8)
    
    c8=(BatchNormalization())(c8)
    u9=(Conv2DTranspose(16,(3,3), strides=(2, 2), activation='relu', kernel_initializer='he_normal', padding='same'))(c8)
    u9=(Dropout(0.3))(u9)
    c9=(Conv2D(16, (3,3),activation='relu', kernel_initializer='he_normal', padding='same'))(u9)
    c9=(Dropout(0.2))(c9)
    
    c9=(BatchNormalization())(c9)
    outputs=(Conv2D(2, (1, 1), activation='sigmoid'))(c9)
    
    model = Model(inputs=[inputs], outputs=[outputs])
    #model.compile(optimizer=optimizer,loss="categorical_crossentropy", metrics=["accuracy"]) 
    return model

**Comment by SuryanarayanaY** [comment]
Hi @feridesecil13 ,

Could you please refer the [reply](https://github.com/keras-team/keras/issues/2588#issuecomment-260615595) here whether serves your purpose. Thanks!
